4001-5000/LIT4089Introduction.xml

```diff
@@ -34,21 +34,25 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       </encodingDesc>
       <profileDesc>
          <abstract>
-            <p/>
+            <p>Chronological summary attributed to <persName ref="PRS4711Giyorgis">Giyorgis Walda ʾAmid</persName>,
+               including a brief chronology of biblical kings, probably an extract from the 
+               <ref type="work" corresp="PRS4711Giyorgi">Tārika Walda ʾAmid</ref>, followed by an outline of Ethiopian kings.
+               It was possibly authored in <date notBefore="1600" notAfter="1705" cert="low">the 17th or begin of the 18th century</date> 
```

Comment:
"It was possibly authored by an unknown Ethiopian author" (?) perhaps to underline that this is a pseudoephigraph.